package.json

```
@@ -1,42 +1,47 @@
 {
-    "name": "homebridge-connexoon",
-    "version": "1.2.0",
-    "description": "A Homebridge plugin providing support for the Connexoon, TaHoma and Cozytouch platforms and accessories.",
-    "main": "index.js",
-    "scripts": {
-        "start": "homebridge -D -U ./homebridge-home -P .",
-        "debug": "ndb npm start",
-        "lint": "eslint ."
-    },
-    "keywords": [
-        "homebridge-plugin",
-        "homebridge",
-        "somfy",
-        "connexoon",
-        "tahoma"
-    ],
-    "author": "Matthieu Di Berardino",
-    "license": "Apache 2",
-    "devDependencies": {
-        "eslint": "^6.1.0",
-        "homebridge": "^0.4.50",
-        "ndb": "^1.1.5"
-    },
-    "engines": {
-        "node": ">=10.0.0",
-        "homebridge": ">=0.4.0"
-    },
-    "dependencies": {
-        "lodash": "^4.17.15",
-        "request": "^2.88.0",
-        "request-promise-native": "^1.0.8"
-    },
-    "homepage": "https://github.com/yenoiwesa/homebridge-connexoon#readme",
-    "repository": {
-        "type": "git",
-        "url": "git://github.com/yenoiwesa/homebridge-connexoon.git"
-    },
-    "bugs": {
-        "url": "http://github.com/yenoiwesa/homebridge-connexoon/issues"
-    }
+  "name": "homebridge-connexoon",
+  "version": "1.3.0",
+  "description": "A Homebridge plugin providing support for the Connexoon, TaHoma and Cozytouch platforms and accessories.",
+  "main": "index.js",
+  "scripts": {
+    "start": "homebridge -D -U ./homebridge-home -P .",
+    "debug": "ndb npm start",
+    "lint": "eslint .",
+    "test": "./node_modules/jest/bin/jest.js"
```

just doing this should work?
```suggestion
    "test": "jest"
```

---

This is true, but only if jest is installed globally. This is going to break the docker file, though.